packages/obonode/obojobo-sections-assessment/components/full-review/index.js

```diff
@@ -36,13 +36,19 @@ class AssessmentReviewView extends React.Component {
 			allScoreDetails: attempts.map(a => a.scoreDetails)
 		})
 
-		const attemptReviewComponent = (attempt, isAHighestScoringNonNullAttempt) => {
+		const attemptReviewComponent = (attempt, isNonNullAttempt) => {
 			const date = new Date(attempt.completedAt)
 			const dateString = formatDate(date, "M/dd/yy 'at' h:mmaaaa")
 			const machineDateString = formatDate(date, "yyyy-MM-dd'T'HH:mm:ss.SSSxxx")
 			const ariaDateString = formatDate(date, "MMMM Do yyyy 'at' h:mmaaaa")
 			const numCorrect = AssessmentUtil.getNumCorrect(attempt.result.questionScores)
 			const numPossibleCorrect = AssessmentUtil.getNumPossibleCorrect(attempt.result.questionScores)
+			const highestAttempt = Math.max.apply(
```

Comment:
I don't think this is still necessary since `highestAttempts` from earlier still already has computed the highest attempt. That calculation is already handled by https://github.com/ucfopen/Obojobo/blob/f54e51cd9937359d3cc6c698f988f63673b9b208/packages/obonode/obojobo-sections-assessment/components/full-review/index.js#L150 and is what is passed in to the argument that was `isAHighestScoringNonNullAttempt`. I'm thinking revert the argument for this function back to `isAHighestScoringNonNullAttempt` or move the logic that I linked to to exist inside this function

Reply:
Jeez. That's simpler than I imagined. I hope it all works now! :)